backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -205,11 +205,9 @@ public String getResultID() {
 		return testEvent.getInternalId().toString();
 	}
 
-	@JsonProperty("Testing_lab_specimen_ID")
-	public String getTestingLabSpecimenID() {
-		// When we grab the device id it should go here
-		// Example: the id on the BinaxNow card
-		return "";
+	@JsonProperty("Test_result_status")
+	public String getTestResultStatus() {
+		return "F";
```

Comment:
Can we start collecting these hard-codes as constants in the top of the file? It's unnerving having to go through line by line to find out what they are.